test.js

```diff
@@ -0,0 +1,78 @@
+require( 'sugar' );
```

Comment:
Again seems like an unnecessary dependency simply for using `.assign` (and for something as simply as converting to string... not even "heavy" string formatting.

When doing auxiliary modules try to keep dependencies to a minimum so they're as thin and lightweight as possible. Its even worth it sometimes to implement your own stuff just to have less dependencies.